sql/src/test/java/io/crate/testing/SQLTransportExecutor.java

```diff
@@ -113,19 +113,19 @@ public void setDefaultSchema(String defaultSchema) {
     }
 
     public SQLResponse exec(String statement) {
-        return executeTransportOrJdbc(false, false, statement, null, REQUEST_TIMEOUT);
+        return executeTransportOrJdbc(false, false, false, statement, null, REQUEST_TIMEOUT);
```

Comment:
Just a suggestion: maybe wrap the 3 params (jdbc, semi, hash) for the `SQLTransportExecutor` in a class, so we don't continue to "explode" the signatures when we add more features in the future.